### DISCUSSION TITLE
Food for thought: How should yarn work based on PRJ_CACHE_HOME

### DISCUSSION THREAD

**Discussion by gytis-ivaskevicius** [numtide/prj-spec#7] [Q&A]
The way I understand `node_modules` should be under `$PRJ_CACHE_HOME` but yarn is able to symlink node modules to reduce the amount of duplicate modules between projects

Does this mean that this `PRJ_CACHE_HOME` is not supported for this use-case or should only symlinks be under `PRJ_CACHE_HOME`?

**Comment by blaggacao** [comment]
I think it is ok to use both, symlinks and files, under any `PRJ_` directory. I would see any constraints.

The fact that a cache manager deduplicates using symlink does not alter the project's view into that cache.

It's transparent.
